src/ucp/api/ucp.h

```diff
@@ -277,6 +278,14 @@ enum ucp_ep_params_flags_field {
                                                            send to a particular
                                                            remote endpoint, for
                                                            example stream */
+    UCP_EP_PARAMS_FLAGS_SEND_CLIENT_ID = UCS_BIT(2)   /**< Send endpoint's worker id
```

Comment:
it is worker client id now, not a worker id, right? Saying just worker id looks confusing a bit